.github/workflows/linter.yml

```diff
@@ -0,0 +1,51 @@
+###########################
```

Comment:
I'd remove all these boilerplate comments, since all explain what the action does but that's all in gh actions docs.

Reply:
Removed. I just kept the link to the documentation.